vitrina/structure/templates/vitrina/structure/dataset_structure.html

```diff
@@ -191,12 +192,12 @@ <h4 class="custom-title is-size-4-mobile">{% translate "Modeliai" %}</h4>
                         {% endif %}
                     </div>
                     <div class="column is-1">
-                        {% if metadata.metadata_version and metadata.draft %}
-                            <span class="tag is-danger is-light">{{ metadata.metadata_version}} ({% translate "rengiama" %})</span>
-                        {% elif metadata.metadata_version and not metadata.draft %}
-                            <span class="tag is-success is-light">{{ metadata.metadata_version }}</span>
-                        {% elif not metadata.metadata_version and metadata.draft %}
+                        {% if metadata.metadata_version.status == "DRAFT" %}
```

Comment:
Shouldn't there also be a check here for metadata.metadata_version as it was before? The following conditions have it


```suggestion
                        {% if metadata.metadata_version and metadata.metadata_version.status == "DRAFT" %}
```

Reply:
Added an if statement on top that checks if metadata.metadata_version is truthy